Lib/http/cookies.py

```diff
@@ -489,6 +489,9 @@ def __set(self, key, real_value, coded_value):
         dict.__setitem__(self, key, M)
 
     def __setitem__(self, key, value):
+        """Check if the key already exists, return without overwriting"""
```

Comment:
It may be beneficial to mention that it's an RFC requirement in the code so if other devs that work on this library see why is it done.